README.md

```diff
@@ -158,9 +187,11 @@ Please consult [the documentation](https://github.com/operator-framework/operato
 
 ## Enabling the Developer  perspective in OpenShift
 
-The frontend can check for the presence of the devconsole CRDs using the Kubernetes API.  Check for [the existence of a Custom Resource Definitions](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#list-customresourcedefinition-v1beta1-apiextensions) with name as `gitsources.devconsole.openshift.io`.  If it exists, it will enable the Developer perspective in the Openshift Console.
+The frontend must check for [the presence of the devconsole Custom Resource Definition](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#list-customresourcedefinition-v1beta1-apiextensions) named `gitsources.devconsole.openshift.io` using the Kubernetes API. This CRD
+enables the Developer perspective in the OpenShift Console.
 
-Refer to OLM test [README](test/README.md) to install the DevOps Console operator.
+Refer to the OLM test [README](test/README.md) to install the DevOps Console
```

Comment:
nit pick - Developer console instead of DevOps.
Also referring to a test for an installation? Maybe it should have been Refer to the OLM test to test e2e?

Reply:
Modified